pixelmod/src/main/java/io/pixelmod/renderer/DecalRendererBuilder.kt

```diff
@@ -0,0 +1,28 @@
+package io.pixelmod.renderer
+
+import io.pixelmod.repository.IDecalRepository
+
+enum class DecalRendererAlgorithmType {
```

Comment:
Do you think it's better to extract this to its own file? so that Android Studio will treat this class as a class instead of a vanilla kotlin file.